spirv_hlsl.hpp

```diff
@@ -23,6 +23,13 @@
 
 namespace spirv_cross
 {
+struct HLSLVertexAttr
+{
+	uint32_t binding;
```

Comment:
This should be called location to match the API. Bindings are used for textures and buffers, etc.

Reply:
I do see the code around it being called mostly "binding", but that's a bit misleading. I should fix that sometime.